src/main/java/com/exasol/projectkeeper/validators/ChangelogValidator.java

```diff
@@ -48,9 +49,10 @@ private void validateThatChangesFileExists(final Consumer<ValidationFinding> fin
         if (!changesFile.exists()) {
             final String changesFileRelativeToProjectRoot = this.mavenProject.getBasedir().toPath()
                     .relativize(changesFile.toPath()).toString();
-            findingConsumer
-                    .accept(new ValidationFinding("E-PK-20 Could not find '" + changesFileRelativeToProjectRoot + "'.",
-                            getFix(changesFile, changesFileRelativeToProjectRoot)));
+            findingConsumer.accept(new ValidationFinding(
+                    ExaError.messageBuilder("E-PK-20").message("Could not find {{changes file}}.")
+                            .parameter("changes file", changesFileRelativeToProjectRoot).toString(),
```

Comment:
You have here `changes file` and in the next it's `changesFile`.

Maybe it's better to unify them